migrations/3_deploy_nori.js

```diff
@@ -1,101 +1,115 @@
 /* globals artifacts web3 */
-const ENS = require('ethereum-ens');
-const { upgradeToContract } = require('../test/helpers/contracts');
+const bluebird = require('bluebird');
+const {
+  deployOrGetRootRegistry,
+  upgradeAndTransferToMultiAdmin,
+} = require('../test/helpers/contracts');
 const getNamedAccounts = require('../test/helpers/getNamedAccounts');
+const utils = require('../test/helpers/utils');
+const {
+  nori,
+  participantRegistry,
+  crc,
+  participant,
+  supplier,
+  verifier,
+  fifoCrcMarket,
+} = require('../test/helpers/contractConfigs');
 
-const ContractRegistryV0_1_0 = artifacts.require('ContractRegistryV0_1_0');
-const RootRegistryV0_1_0 = artifacts.require('RootRegistryV0_1_0');
-const NoriV0_1_0 = artifacts.require('NoriV0_1_0');
-const ParticipantRegistryV0_1_0 = artifacts.require(
-  'ParticipantRegistryV0_1_0'
-);
-const CRCV0_1_0 = artifacts.require('CRCV0_1_0');
-const ParticipantV0_1_0 = artifacts.require('ParticipantV0_1_0');
-const SupplierV0_1_0 = artifacts.require('SupplierV0_1_0');
-const VerifierV0_1_0 = artifacts.require('VerifierV0_1_0');
-const FifoCrcMarketV0_1_0 = artifacts.require('FifoCrcMarketV0_1_0');
+const MultiAdmin = artifacts.require('MultiAdmin');
+
+const upgradeAndMigrateContracts = (
+  config,
+  adminAccountAddress,
+  contractsToUpgrade,
+  multiAdmin,
+  registry
+) =>
+  bluebird.mapSeries(contractsToUpgrade, async contractConfig => {
```

Comment:
This is how you would do this without bluebird...
```js
for (contractConfig of contractsToUpgrade) {
  const configuredContract = await contractConfig(multiAdmin, registry);
  ...
}
```

Reply:
I think I had a for of loop first, but eslint was screaming at me for putting it here. I'm sure bluebirds doing the same thing under the hood though, and so maybe there's an eslint config I can change instead?